int64 robot_type
float64 width
float64 length
simulation_control/Wheel[] wheels